R/abm.R

```diff
@@ -604,7 +651,8 @@ abm_private_data <- function(unit_code) {
       abm_table3(con = db, unit_code = x),
       abm_table4(con = db, unit_code = x),
       abm_table5(con = db, unit_code = x),
-      publications = abm_publications(con = db, unit_code = x)
+      abm_woscoverage(con = db, unit_code = x),
```

Comment:
this list slot was named in abm_public_data() but not here? ie `coverage = abm_woscoverage(...)`

Reply:
Good catch. Fixing.